parachains/runtimes/collectives/collectives-polkadot/src/weights/pallet_alliance.rs

```diff
@@ -222,33 +244,45 @@ impl<T: frame_system::Config> pallet_alliance::WeightInfo for WeightInfo<T> {
 	// Storage: AllianceMotion Members (r:0 w:1)
 	// Storage: AllianceMotion Prime (r:0 w:1)
 	fn elevate_ally() -> Weight {
-		Weight::from_ref_time(30_236_000 as u64)
+		Weight::from_ref_time(36_595_000 as u64)
 			.saturating_add(T::DbWeight::get().reads(4 as u64))
 			.saturating_add(T::DbWeight::get().writes(4 as u64))
 	}
-	// Storage: Alliance UpForKicking (r:1 w:0)
-	// Storage: Alliance Members (r:3 w:1)
+	// Storage: Alliance Members (r:4 w:2)
 	// Storage: AllianceMotion Proposals (r:1 w:0)
-	// Storage: Alliance DepositOf (r:1 w:1)
-	// Storage: System Account (r:1 w:1)
 	// Storage: AllianceMotion Members (r:0 w:1)
 	// Storage: AllianceMotion Prime (r:0 w:1)
-	fn retire() -> Weight {
-		Weight::from_ref_time(44_311_000 as u64)
-			.saturating_add(T::DbWeight::get().reads(7 as u64))
+	// Storage: Alliance RetiringMembers (r:0 w:1)
+	fn give_retirement_notice() -> Weight {
+		Weight::from_ref_time(39_034_000 as u64)
+			.saturating_add(T::DbWeight::get().reads(5 as u64))
 			.saturating_add(T::DbWeight::get().writes(5 as u64))
 	}
+	// Storage: Alliance RetiringMembers (r:1 w:1)
+	// Storage: Alliance Members (r:1 w:1)
+	// Storage: Alliance DepositOf (r:1 w:1)
+	// Storage: System Account (r:1 w:1)
+	fn retire() -> Weight {
+		Weight::from_ref_time(43_383_000 as u64)
+			.saturating_add(T::DbWeight::get().reads(4 as u64))
+			.saturating_add(T::DbWeight::get().writes(4 as u64))
+	}
 	// Storage: Alliance Members (r:3 w:1)
 	// Storage: AllianceMotion Proposals (r:1 w:0)
 	// Storage: Alliance DepositOf (r:1 w:1)
-	// Storage: System Account (r:1 w:1)
+	// Storage: System Account (r:2 w:2)
+	// Storage: ParachainInfo ParachainId (r:1 w:0)
+	// Storage: PolkadotXcm SupportedVersion (r:1 w:0)
+	// Storage: PolkadotXcm VersionDiscoveryQueue (r:1 w:1)
+	// Storage: PolkadotXcm SafeXcmVersion (r:1 w:0)
+	// Storage: ParachainSystem HostConfiguration (r:1 w:0)
+	// Storage: ParachainSystem PendingUpwardMessages (r:1 w:1)
 	// Storage: AllianceMotion Members (r:0 w:1)
 	// Storage: AllianceMotion Prime (r:0 w:1)
-	// Storage: Alliance UpForKicking (r:0 w:1)
 	fn kick_member() -> Weight {
-		Weight::from_ref_time(46_112_000 as u64)
-			.saturating_add(T::DbWeight::get().reads(6 as u64))
-			.saturating_add(T::DbWeight::get().writes(6 as u64))
+		Weight::from_ref_time(122_258_000 as u64)
+			.saturating_add(T::DbWeight::get().reads(13 as u64))
+			.saturating_add(T::DbWeight::get().writes(8 as u64))
```

Comment:
changed significantly.
from the Storage docs we see this time the teleport included in the beanchmark.